interfaces/Portalicious/src/app/services/auth/strategies/basic-auth/basic-auth.login.component.html

```diff
@@ -0,0 +1,46 @@
+<app-form-default
+  [formGroup]="formGroup"
+  submitButtonText="Log in"
+  i18n-submitButtonText
+  [mutation]="loginMutation"
+>
+  <app-form-field-wrapper
+    label="E-mail"
+    i18n-label
+    [errorMessage]="formFieldErrors()('email')"
+  >
+    <!-- eslint-disable @angular-eslint/template/no-autofocus -->
+    <input
+      pInputText
+      type="email"
+      formControlName="email"
+      autocomplete="email username"
+      placeholder="yourname@example.org"
+      i18n-placeholder
+      pAutoFocus
+      autofocus
+    />
+  </app-form-field-wrapper>
+  <app-form-field-wrapper
+    label="Password"
+    i18n-label
+    [errorMessage]="formFieldErrors()('password')"
+  >
+    <p-password
+      formControlName="password"
+      autocomplete="current-password"
+      [toggleMask]="true"
+      [feedback]="false"
+    />
+  </app-form-field-wrapper>
+</app-form-default>
+<p class="mt-4">
+  <span i18n>Forgot your password?</span>&nbsp;
+
+  <a
+    i18n
+    href="mailto:support@510.global?subject=Request Password reset for 121 Platform"
```

Comment:
Sanity-check with @tijsziere : Don't we want to use `support@121.global` everywhere?

Reply:
I followed the design, but we can add it back to the main page when we implement SSO if we have to.

Reply:
Ah sorry, I misunderstood the original comment. I will fix this in another PR.